spec/features/user_spec.rb

```diff
@@ -69,7 +69,7 @@
 
   it 'not allows to create user with wrong data' do
     visit sign_up_path
-    click_button 'Регистрация'
+    click_button 'Зарегистрироваться'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.